packages/datagrid/src/components/DataGrid/__snapshots__/DataGrid.test.js.snap

```diff
@@ -265,8 +265,9 @@ exports[`#DataGrid should render one Skeleton is InProgress 1`] = `
 <div
   className="theme-td-grid-loading theme-td-grid td-grid"
 >
-  <Translate(Skeleton)
```

Comment:
Have you do yarn on @talend/ui root to refresh components in the datagrid ( with the yarn package ) it seems weird to rollback this translations